qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -383,13 +383,14 @@ def _transpiled_circuits(self) -> List[QuantumCircuit]:
         # Compute average basis gate numbers per Clifford operation
         # This is probably main source of performance regression.
         # This should be integrated into transpile pass in future.
+        qubit_indices = {bit: index for index, bit in enumerate(transpiled[0].qubits)}
         for circ in transpiled:
```

Comment:
All transpiled circuits has qubits with unique name "q" so no problem in these lines, but a bit slower
```
for circ in transpiled:
    qubit_indices = {bit: index for index, bit in enumerate(circ.qubits)}
```
might be safer.

Reply:
but all RB circuits have the same qubits no? otherwise i think a test should have failed